src/pages/editprofile.jsx

```diff
@@ -6,190 +6,294 @@ import Button from "@/components/ui/Button";
-                            <Input
-                                inputWidthSize='flex-[2_1_0%]'
-                                type='text'
-                                name='fullName'
-                                label={t("fullName")}
-                                labelColor='text-black'
-                                value={formData.fullName || ""}
-                                onChange={handleChange}
-                            />
-                        </div>
-                        <div className='flex items-center my-5'>
-                            <Dropdown
-                                placeholder='select'
-                                data={[
-                                    { value: 1, label: t("bacholar") },
-                                    { value: 2, label: t("master") },
-                                    { value: 3, label: t("PhD") },
-                                    { value: 4, label: t("deploma") },
-                                ]}
-                                label={t("educationLevel")}
-                                labelColor='text-black'
-                                value={formData.educationLevel || ""}
-                                onChange={handleChange}
-                            />
-                        </div>
-                        <div className='flex items-center my-5'>
-                            <Input
-                                inputWidthSize='flex-[2_1_0%]'
-                                type='text'
-                                name='hubbies'
-                                label={t("hubbies")}
-                                labelColor='text-black'
-                                value={formData.hubbies || ""}
-                                onChange={handleChange}
-                            />
-                        </div>
-                        <div className='flex items-center my-5'>
-                            <Input
-                                inputWidthSize='flex-[1_1_0%]'
-                                type='number'
-                                name='familySize'
-                                label={t("familySize")}
-                                labelColor='text-black'
-                                value={formData.familySize || ""}
-                                onChange={handleChange}
-                            />
-                            <div className='ms-10 flex-[0_1_0%]'>
-                                {t("member(s)")}
+            {authenticated ? (
+                <div className='grid grid-cols-1 lg:grid-cols-2 py-20 gap-y-10'>
+                    <div className='justify-self-center lg:justify-self-start'>
+                        <PreviewProfile />
+                    </div>
+                    <div className='w-full justify-self-center lg:justify-self-end'>
+                        <PageIntro title={t("profileInfo")} />
```

Comment:
Dont replace `<fieldset>` & `<legend>` by pageintro, its set carefully to group elements that are related, moreover, set according to properties of theme, so undo here and other all places

Reply:
This pull request is still a work in progress